.github/workflows/publish_intellij_plugin.yml

```diff
@@ -0,0 +1,43 @@
+name: Build and Publish Intellij Plugin
+
+on:
+  push:
+    branches:
+      - prod-sync
+    paths:
+      - '_data/latest/**'
+    
+jobs:
+  trigger_workflow:
+     strategy:
+       matrix:
+         os: [ubuntu-latest]
+     runs-on: ${{ matrix.os }}
+     
+     env:
+         working-directory: ./tool-plugins/intellij
+         publish_token: ${{ secrets.IDEA_TOKEN }}
+
+     steps:
+       - uses: actions/setup-java@v1
+         with:
+           java-version: 1.8
+           
+       - name: Install jq for json manipulation
+         run: sudo apt-get install jq
+         
+       - name: Get ballerina version
+         run: |
+              version="`jq -r '.version' _data/latest/metadata.json`"
+              BAL_VERSION="v${version}"
+
+       - uses: actions/checkout@v2
+         with:
+           repository: ballerina-platform/ballerina-lang
+           ref: $BAL_VERSION
+           
+       - name: Build and publish plugin
+         working-directory: ${{env.working-directory}}
```

Comment:
Instead of setting it a environment variable and getting it back here. Can't we directly set the value here.